src/components/search-form/search-form.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Button from '../common/button/button';
+import Input from '../common/input/input';
+import PropTypes from 'prop-types';
+import './search-form.scss';
+
+export default class SearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      searchQuery: this.props.initialQuery
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleFormSubmit = this.handleFormSubmit.bind(this);
```

Comment:
I believe that instead of using binding it's more readable and is more modern to use arrow functions there (as you did in counter component)

Reply:
fixed in https://github.com/SiarheiDrozd/ReactJS/pull/5